src/Faker/Provider/bg_BG/Person.php

```diff
@@ -10,78 +10,95 @@ class Person extends \Faker\Provider\Person
     );
 
     protected static $maleNameFormats = array(
-        '{{firstNameMale}} {{lastNameMale}}',
```

Comment:
please don't remove these duplicates, there are here to provide balance between the common and uncommon formats.